packages/account/src/Configs/personal-details-config.ts

```diff
@@ -210,9 +209,7 @@ export const personal_details_config = ({
         },
     };
 
-    // Need to check if client is high risk (only have SVG i.e. China & Russia)
-    // No need to get additinal details when client is high risk
-    if (!is_high_risk_client_for_mt5 && real_account_signup_target !== 'maltainvest') {
+    if (real_account_signup_target !== 'maltainvest') {
```

Comment:
I think we can remove the usage of is_high_risk_client_for_mt5 from account-wizard.jsx